elide-model-config/src/test/resources/validator/valid/models/tables/player_stats.hjson

```diff
@@ -14,7 +14,7 @@
       category: Table Category
       cardinality : lARge
       hidden : false
-      readAccess : A user is admin or is a player in the game
+      readAccess : (Principal is user AND Principal is member) OR (Principal is admin AND NOT Principal is guest)
```

Comment:
I"m not sure how friendly this is to expect HJSON configuration to type in "Principal is" for each of the defined roles.  We should discuss this.